src/proposal.mligo

```diff
@@ -79,6 +79,18 @@ let is_le_qt(qt_1, qt_2 : quorum_threshold * quorum_threshold): bool =
   let (nat_1, nat_2) = cmp_qt (qt_1, qt_2) in
   nat_1 <= nat_2
 
+// Returns the greatests of two given quorum_thresholds
+let max_qt (a, b : quorum_threshold * quorum_threshold) : quorum_threshold =
+  if is_gt_qt(a, b)
+  then a
+  else b
+
+// Returns the greatests of two given quorum_thresholds
+let min_qt (a, b : quorum_threshold * quorum_threshold) : quorum_threshold =
+  if is_gt_qt(a, b)
+  then b
+  else a
```

Comment:
These aren't really used by this module and I think it's a bit odd to see them here.

I suggest moving them to `defaults` (no change in the importing will be needed) and make a single `bound_qt` (see previous comment).